server/models.js

```diff
@@ -24,6 +24,27 @@ const User = sequelize.define('user', {
 	location: Sequelize.STRING
 });
 
+const Booking = sequelize.define('booking', {
+	helperId: Sequelize.STRING,
+	clientId: Sequelize.STRING,
+	skill: Sequelize.STRING,
+	created: Sequelize.INTEGER,
+	serviceDetails: Sequelize.STRING,
+	price: Sequelize.FLOAT,
+	location: Sequelize.String,
+});
+
+const Listing = sequelize.define('booking', {
+	clientId: Sequelize.STRING,
+	skill: Sequelize.STRING,
+	created: Sequelize.STRING,
+	active: Sequelize.BOOLEAN,
+	serviceDetails: Sequelize.BOOLEAN,
```

Comment:
Let's make `serviceDetails` a STRING and change `active` to `isActive` to match the user property convention.